### PR TITLE
KOGITO-480 - Upgrade Quarkus to version 0.27.0

### PR DESCRIPTION
**Pull request by cristianonicolai**
Assembly: 
- https://github.com/kiegroup/kogito-runtimes/pull/209
- https://github.com/kiegroup/kogito-bom/pull/65
- https://github.com/kiegroup/kogito-examples/pull/100

### CODE REVIEW

**Review by mswiderski — APPROVED** [review]

**Comment by mswiderski** [comment]
@ricardozanini just so you're aware

File: pom.xml [hunk 1]
```diff
@@ -67,12 +67,12 @@
...
     <version.javax.annotation-api>1.3.2</version.javax.annotation-api>
 
-    <version.org.infinispan>10.0.0.Beta5</version.org.infinispan>
-    <version.org.infinispan.protostream>4.3.0.Alpha9</version.org.infinispan.protostream>
+    <version.org.infinispan>10.0.0.CR1</version.org.infinispan>
+    <version.org.infinispan.protostream>4.3.0.Alpha10</version.org.infinispan.protostream>
 
-    <version.io.quarkus>0.23.2</version.io.quarkus>
+    <version.io.quarkus>0.27.0</version.io.quarkus>
     <version.com.oracle.substratevm>19.2.0.1</version.com.oracle.substratevm>
```
Comment (ricardozanini) [inline pom.xml:74]: @cristianonicolai @mswiderski 0.27.0 requires GraalVM 19.2.1
https://quarkus.io/blog/quarkus-0-27-0-released/

Comment (cristianonicolai) [reply pom.xml:74]: @ricardozanini I looked into that and apparently it takes a while for the Maven artifacts to be published. See https://github.com/quarkusio/quarkus/blob/master/bom/runtime/pom.xml#L77

Comment (mswiderski) [reply pom.xml:74]: though quarkus recommends to use 19.2.1 as part of 0.27 release https://quarkus.io/blog/quarkus-0-27-0-released/

Comment (cristianonicolai) [reply pom.xml:74]: @mswiderski correct, its just that we can't update the pom.xml until the artifacts are uploaded. That's the same issue that Quarkus itself has.

**Pull request merged**